README.md

```diff
@@ -6,6 +6,27 @@ Mosquito Defenders is our entry for the "1UP for NASA Earth!" challenge in the [
 
 In the game, we imagine what it might be like if citizen scientists were the last line of defense against mosquito-borne diseases like malaria. You play as the head of a public health agency tasked with responding to mosquito sighting reports from these citizen scientists. You have a limited budget of money and people available, and you'll need to spread them over the globe to stall or prevent disease outbreaks.
 
+## How to run
+### Requirements
+This game was made using Unity, with a python web server serving mosquito outbreak events. You will need the following installed:
+ - Unity
+ - Python3
+ - pipenv
+
+To set up the server, run the following commands.
+```
+cd Data
+pipenv --python3.7 install
+```
+
+### Running
+To run the server, run the following from the `Data/` directory.
+```
+pipenv run ./flask_app.py
```

Comment:
```suggestion
pipenv make serve
```
gets you a development server with live reload, so you can change flask_app.py and have changes apply instantly